vite.config.mts

```diff
@@ -148,7 +161,7 @@ export default defineConfig({
     dedupe: ['react', 'ethers', 'react-dom', 'native-base', 'bn.js'],
   },
   build: {
-    sourcemap: !!process.env.SENTRY_AUTH_TOKEN ? 'hidden' : false, //required for sentry
+    sourcemap: !process.env.SENTRY_AUTH_TOKEN ? 'hidden' : false, //required for sentry
```

Comment:
linter flagged it as redundant

Reply:
not true

Reply:
![image](https://github.com/GoodDollar/GoodDAPP/assets/6606028/353398dd-2aba-42a2-86eb-45ea5f9fa62d)